src/header.js

```diff
@@ -34,6 +34,7 @@ try {
   var incrementalmh = require('./inference/incrementalmh');
   var optimize = require('./inference/optimize');
   var forwardSample = require('./inference/forwardSample');
+  var interleavingSF = require('./inference/interleavingSF');
```

Comment:
I'd rename `interleavingSF` to `checkSampleAfterFactor`, or something similar that is more explicit about what the coroutine does.